fitbenchmarking/parsing/parse.py

```diff
@@ -38,8 +40,11 @@ def parse_problem_file(prob_file):
     """
 
     prob_type = determine_problem_type(prob_file)
-    logger.info("*** Loading {0} formatted problem definition "
-                "file {1} ***".format(prob_type, prob_file))
+    # print(os.path.basename(prob_file))
```

Comment:
please remove

```
    # print(os.path.basename(prob_file))
    # logger.info("Loading {0} formatted problem definition file {1} | Path: "
    #             "{2}".format(prob_type,prob_file.rsplit('/',1)[1],prob_file[prob_file.find('fitbenchmarking'):]))
```

no need since you can always restore any old code from git history

Reply:
I will remove them accordingly.